spec/controllers/enrollments_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe EnrollmentsController, type: :controller do
+  let(:white_player) { FactoryGirl.create(:player, email: 'blah@blah.com', password: 'SPACECAT') }
+  let(:black_player) { FactoryGirl.create(:player, email: 'meow@meow.com', password: 'MONORAILCAT') }
+  let(:game) { FactoryGirl.create(:game, white_player_id: white_player.id, black_player_id: nill) }
```

Comment:
Thanks @kellylougheed - I fixed the typo (actually, set both to `0` to reflect your code) and pushed. That said, I'm still having the same issue. The holiday weekend has been brutal for me, so I'll be back on this tonight after getting some fresh air 😃